src/implementations/utils/__tests__/wallet.test.ts

```diff
@@ -42,7 +42,7 @@ describe("wallet", () => {
     });
     expect(wallet.address).toStrictEqual(walletAddress);
     expect(wallet.privateKey).toStrictEqual(privateKey);
-  });
+  }, 20000);
```

Comment:
From our conv, CircleCI is timing out at 5s and the execution time is around 11s. Suggested for you to increase the global timeout for jest: https://jestjs.io/docs/en/configuration#testtimeout-number